docs/source/developers/python.rst

```diff
@@ -379,8 +379,23 @@ debugging a C++ unitttest, for example:
 Building on Windows
 ===================
 
-First, we bootstrap a conda environment similar to above, but skipping some of
-the Linux/macOS-only packages:
+.. warning::
+   Building on Windows is currently broken. The issue is being worked on
+   under the `Github PR #5247 <https://github.com/apache/arrow/pull/5247>`_.
```

Comment:
Please remove this. Now that https://issues.apache.org/jira/browse/ARROW-6457 it's building fine.